### PR TITLE
Add application environment variables & husky error on Mac

### PR DESCRIPTION
**Pull request by shpark76**
- Added application environment variables for frontend app
- Fixed git commit error(husky error) on Mac

### CODE REVIEW

File: wherehows-frontend/application.env [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Secret Key
+WHZ_SECRET="change_me"
+
+# Database Connection
+WHZ_DB_DRIVER="com.mysql.jdbc.Driver"
+WHZ_DB_NAME="wherehows"
+WHZ_DB_USERNAME="wherehows"
+WHZ_DB_PASSWORD="wherehows"
+
+# Fully qualified jdbc url
```
Comment (alyiwang) [inline wherehows-frontend/application.env]: For frontend, don't require this config

File: wherehows-frontend/application.env [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Directory containing kerberos config files
+WHZ_KRB5_DIR="/dir/to/krb5conf"
```
Comment (alyiwang) [inline wherehows-frontend/application.env]: For frontend, don't require this config, see wherehows-frontend/conf/application.conf

File: wherehows-frontend/application.env [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Secret Key
+WHZ_SECRET="change_me"
+
+# Database Connection
+WHZ_DB_DRIVER="com.mysql.jdbc.Driver"
```
Comment (alyiwang) [inline wherehows-frontend/application.env]: WHZ_DB_DRIVER not needed

File: wherehows-frontend/application.env [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+# Serach Engine
+WHZ_SEARCH_ENGINE=elasticsearch
+
+# Elasticsearch (Change "localhost" to your Es host )
+WHZ_ES_DATASET_URL="http://localhost:9200/wherehows/dataset/_search"
+WHZ_ES_METRIC_URL="http://localhost:9200/wherehows/metric/_search"
+WHZ_ES_FLOW_URL="http://localhost:9200/wherehows/flow_jobs/_search"
+
+# Directory containing ETL job files
+WHZ_ETL_JOBS_DIR="/var/tmp/jobs"
```
Comment (alyiwang) [inline wherehows-frontend/application.env]: Not needed for frontend

File: wherehows-frontend/application.env [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+# Temp directory for ETL job
+WHZ_ETL_TEMP_DIR="/var/tmp/wherehows"
+
+# LDAP
+WHZ_LDAP_URL=your_ldap_url
+WHZ_LDAP_PRINCIPAL_DOMAIN=your_ldap_principal_domain
+WHZ_LDAP_SEARCH_BASE=your_ldap_search_base
+
+# Piwik tracking configuration
+PIWIK_SITE_ID="0000" # change_to_your_piwik_id
```
Comment (alyiwang) [inline wherehows-frontend/application.env]: Piwik might be a linkedin internal feature only.

File: wherehows-frontend/runFrontend [hunk 6]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -a
+source ./application.env
+set +a
+
+export JAVA_OPTS="-Xms512m -Xmx2048m -Dhttp.port=19001"
```
Comment (alyiwang) [inline wherehows-frontend/runFrontend]: port 9001

**Comment by alyiwang** [comment]
Might need to fix travis ci

File: wherehows-frontend/application.env (hunk 2 above)
Comment (shpark76) [inline wherehows-frontend/application.env]: applied

File: wherehows-frontend/application.env (hunk 3 above)
Comment (shpark76) [inline wherehows-frontend/application.env]: applied

File: wherehows-frontend/application.env (hunk 1 above)
Comment (shpark76) [inline wherehows-frontend/application.env]: will keep it.

File: wherehows-frontend/application.env (hunk 4 above)
Comment (shpark76) [inline wherehows-frontend/application.env]: removed Directory containing ETL job files section

File: wherehows-frontend/application.env (hunk 5 above)
Comment (shpark76) [inline wherehows-frontend/application.env]: will keep it.

File: wherehows-frontend/runFrontend (hunk 6 above)
Comment (shpark76) [inline wherehows-frontend/runFrontend]: changed to 9001

**Pull request merged**